defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+xen_vman_vms:                   []
+xen_vman_memory:                1024
+xen_vman_vcpus:                 2
+xen_vman_auto_boot:             1
```

Comment:
Is this a boolean value? Please use `True` in that case. Also for the other variables.